packages/react/src/components/Hero/Hero.tsx

```diff
@@ -58,7 +58,9 @@ const Hero: FC<HeroProps> = ({
 
   return (
     <div className={heroClasses}>
-      {image ? <HeroImage {...image} /> : <div className="hero--figure" />}
+      <div className="hero--figure">
+        {image ? <HeroImage {...image} /> : null}
```

Comment:
You can do a simpler image && <HeroImage /> here

Reply:
Doing an && would return 0 so as it returns true or false so i think ternary operator would be the best in this case

Reply:
Made the change :)

Reply:
Only if image is a number and its zero 😁